lib/active_record/connection_adapters/oracle_enhanced/schema_creation.rb

```diff
@@ -5,10 +5,12 @@ class SchemaCreation < AbstractAdapter::SchemaCreation
         private
 
           def visit_ColumnDefinition(o)
-            if [:blob, :clob].include?(sql_type = type_to_sql(o.type.to_sym,  o.options).downcase.to_sym)
-              if (tablespace = default_tablespace_for(sql_type))
-                @lob_tablespaces ||= {}
-                @lob_tablespaces[o.name] = tablespace
+            unless o.type.nil?
+              if [:blob, :clob].include?(sql_type = type_to_sql(o.type.to_sym,  o.options).downcase.to_sym)
```

Comment:
I think that `to_sym` to `o.type` is unnecessary.
https://github.com/rails/rails/blob/55d66e214825b89442383f90adf575cf8c96568c/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L1056